include/dylib.hpp

```diff
@@ -11,6 +11,7 @@
 
 #pragma once
 
+#include <iostream>
```

Comment:
Did you intend to add iostream here? Not sure it makes sense.

Reply:
I forgot to remove it after some debug.
Done